docs/developer-docs/multi-chain/ethereum/using-eth/eth-dev-workflow.mdx

```diff
@@ -0,0 +1,62 @@
+---
+keywords: [advanced, ethereum, addresses, eth address ]
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Generating addresses
+
+<MarkdownChipRow labels={["Advanced", "Ethereum" ]} />
+
+## Overview
+
+Developers on ICP can create canisters and dapps that interact with the Ethereum mainnet, Sepolia testnet, and other EVM-compatible networks through ICP's Ethereum integration. To use this integration, developers must implement a specific workflow that makes JSON-RPC calls to the EVM RPC canister in order to query network information and submit transactions to an EVM-compatible network. Developers can also integrate and interact with chain-key Ethereum (ckETH) from within a canister, which uses the ICRC-1 token implementation.
+
+## Using the Ethereum integration
+
+To interact with the Ethereum network or another EVM-compatible network, a developer will need to use a combination of the management canister API and EVM RPC canister RPC methods to complete the following steps:
+
+1. [Generate a public key](generating-addresses.mdx): To sign and submit transactions, your canister will need a public key. You will also need this public key to generate an Ethereum wallet address.
+
+2. [Sign a transaction](signing-transactions.mdx): Next, a transaction must be signed before it can be submitted to the Ethereum network and executed. Transactions are signed using a canister's public key.
+
+3. [Submit a transaction](submit-transactions.mdx): Signed raw transactions can be submitted using the EVM RPC canister's RPC method `eth_sendRawTransaction`. Code examples for sending transactions can be found in the [EVM RPC canister](/docs/current/developer-docs/multi-chain/ethereum/evm-rpc/evm-rpc-canister) documentation.
+
+4. [Read information from the Ethereum network](/docs/current/developer-docs/multi-chain/ethereum/evm-rpc/evm-rpc-canister): Information from the Ethereum network can be queried using RPC methods such as:
+
+    - `eth_getBlockByNumber`: Returns information about a given block.
+
+    - `eth_getTransactionCount`: Returns the number of transactions for a specified address.
+
+    - `eth_getTransactionReceipt`: Returns details about a submitted transaction.
+
+    - `eth_feeHistory`: Returns the historical fee data to estimate gas prices for transactions.
+
+    - `eth_getLog`s: Returns the logs of a specified block or transaction.
+
+Each of these steps can be executed programmatically within a smart contract that is deployed on ICP. View the [EVM RPC canister](/docs/current/developer-docs/multi-chain/ethereum/evm-rpc/evm-rpc-canister) documentation to see code examples for each RPC method.
+
+## Integrating with Ethereum
```

Comment:
If we remove the EVM RPC note, maybe we can make this section about SIWE wallet authentication only?

Reply:
Removed EVM RPC info and added additional SIWE info